core/tests/Drupal/Nightwatch/Tests/index.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   'Demo Drupal.org': (browser) => {
     browser
-      .url('https://www.drupal.org/')
+      .drupalURL('/community')
```

Comment:
What about using "relativeUrl" or something like that? Using Drupal makes me shrug here.

Reply:
Good point. I'll change it.